### ISSUE TITLE
NVDA Speech viewer is showing the title of a dialogue but it is not announcing it

### ISSUE THREAD

**Issue by NitinPem** [nvaccess/nvda#13558]
I have a a weird situation. NDVA speech viewer is showing a dialogue title. But it is not being announced by NVDA
Have anyone ever come across this situation? Please refer to attached. The highlighted part is showing in the NVDA speech viewer but not being announced by NVDA. It only announces 'Create' then it jumps to Reason combo Box.

I have a feeling that this may be a bug within NVDA. Version is:  2021.3.1
![2022-03-28 21_14_51-Window](https://user-images.githubusercontent.com/102540782/160635017-04d1c46f-d5ed-47a9-ac3a-0f6fda722bec.png)

**Comment by XLTechie** [comment]
1. Why have you opened multiple issues for this?

2. Please post an NVDA log at debug level showing this happening.

To do that:
Start NVDA.
Go to general settings.
Set the log level to debug.
Press OK.

Turn on the speech viewer.
Do whatever causes this problem.

Once the problem has happened, attach to a comment on this issue, a copy of the 
file at %temp%\nvda.log

Note that if you have typed any sensitive information while NVDA was in debug 
mode, it will appear in the log. You can open the file in notepad if you want to 
look through it first.

3. Please fill out the bug template when posting issues.

4. Please remember that most people who might help you on this project are 
blind. A screen shot is not going to help us help you. We need debug logs 
anyway, but for this reason as well.

**Comment by seanbudd** [comment]
Please note that images are not accessible to vision impaired developers.
I'm closing this issue. Please reopen a new issue following @XLTechie's advice.
